README.md

```
@@ -1,26 +1,152 @@
+pnpm build:pre
+
+# Run post-build operations
+pnpm build:post
+
+# Run linting
+pnpm lint
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)